WooCommercePOS/WooCommercePOS/Utils/Color+WooCommercePOS.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+extension Color {
+
+    /// Default POS background color
+    ///
+    static var posBackground: Color {
+        return Color(red: 11, green: 146, blue: 213)
+    }
+
+    /// Default POS foreground color
+    ///
+    static var posForeground: Color {
+        return Color(red: 89, green: 181, blue: 227)
+    }
+
+    /// Default POS fields color
+    ///
+    static var posFields: Color {
+        return Color(red: 142, green: 208, blue: 240)
+    }
+
+    /// Default POS text color
+    ///
+    static var posText: Color {
+        return Color.primary
+    }
```

Comment:
Since the file is already located in the POS target and namespace, wdyt if we remove the `pos*` prefix?
```suggestion
    static var background: Color {
        return Color(red: 11, green: 146, blue: 213)
    }

    /// Default POS foreground color
    ///
    static var foreground: Color {
        return Color(red: 89, green: 181, blue: 227)
    }

    /// Default POS fields color
    ///
    static var fields: Color {
        return Color(red: 142, green: 208, blue: 240)
    }

    /// Default POS text color
    ///
    static var primaryText: Color {
        return Color.primary
    }
```